JASP-Engine/JASP/R/descriptives.R

```diff
@@ -117,6 +117,23 @@ Descriptives <- function(jaspResults, dataset, options) {
       }
     }
   }
+  # Interval plots
+  if (options$descriptivesIntervalPlot) {
+    if(is.null(jaspResults[["IntervalPlots"]])) {
+      jaspResults[["IntervalPlots"]] <- createJaspContainer(gettext("Intervalplots"))
+      jaspResults[["IntervalPlots"]]$dependOn(c("IntervalPlots", "splitby"))
+      jaspResults[["IntervalPlots"]]$position <- 11
```

Comment:
So since the position is 11 (last at the moment), could you also move this code to appear last in the function? So below the scatter plots.